HomeWork/src/main/java/hashmap/FileNavigator.java

```diff
@@ -0,0 +1,54 @@
+package hashmap;
+
+import javax.naming.SizeLimitExceededException;
+import java.io.File;
+import java.lang.module.FindException;
+import java.nio.file.NoSuchFileException;
+import java.util.*;
+
+public class FileNavigator {
+
+    private HashMap<String, List<FileData>> navigator = new HashMap<>();
```

Comment:
Always initialize your collection as follows: the interface is declared on the left, and implementation is declared on the right, e.g. `Map<String, List<FileData>> navigator = new HashMap<>();`